sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SynchronousAnalyzer.java

```diff
@@ -57,7 +57,7 @@ public void waitForDone() {
       if (count % logFrequency == 0) {
         LOGGER.info("Waiting for analysis reports to be integrated");
       }
-      String response = server.post(RELATIVE_URL, Collections.<String, Object>emptyMap());
+      String response = server.post(RELATIVE_URL, Collections.emptyMap());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "post"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)